packages/visage/src/components/Input.tsx

```diff
@@ -0,0 +1,105 @@
+import React, {
+  ReactElement,
+  ReactNode,
+  ComponentPropsWithRef,
+  useMemo,
+} from 'react';
+import { StyleSheet } from '@byteclaw/visage-core';
+import { createComponent } from '../core';
+import { Box } from './Box';
+import { StyleProps } from '../createNPointTheme';
+
+const InputBase = createComponent('input', {
+  displayName: 'InputBase',
+  defaultStyles: {
+    borderWidth: 0,
+    background: 'inherit',
+    color: 'inherit',
+    fontSize: 'inherit',
+    lineHeight: 'inherit',
```

Comment:
Este nie som stale presvedceny o tych inheritoch, ci si nezadelame na problemy s tym riesenim. Je to trosku hack, nie?

Reply:
v čom je to hack, inherit je predsa normálna value. Akurát, že `background` by stačilo nastaviť na `transparent` pretože parent div má background.